docs/providers/aws/guide/iam.md

```diff
@@ -57,7 +57,7 @@ provider:
   iamManagedPolicies:
     - 'some:aws:arn:xxx:*:*'
     - 'someOther:aws:arn:xxx:*:*'
-    - { 'Fn::Join': [':', ['arn:aws:iam:', { Ref: 'AWSAccountId' }, 'some/path']] }
+    - { 'Fn::Join': [':', ['arn:aws:iam:', { Ref: 'AWSA::ccountId' }, 'some/path']] }
```

Comment:
I believe it should be `AWS::AccountId`

Reply:
Thanks for checking! Fixed it. Sorry for making it more complicated than it should have been...